app/src/main/res/layout/dialog_results.xml

```diff
@@ -343,6 +343,7 @@
 
         <include layout="@layout/expanded_results_box"
             android:id="@+id/expandedResultsBox"
+            android:visibility="gone"
```

Comment:
@DJCrapsody So apparently what we're using is U+25B6 and U+25BC. Those are both in the same block as the ones you want. Can you file a separate issue so we can figure out what's going on?